selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolder.java

```diff
@@ -84,7 +84,12 @@ public void factoryWithDirectUrlWithRootFolder() throws Exception {
             "tslint.json",
             "typings.json",
             "wallaby.js");
-    testFactoryWithRootFolder.authenticateAndOpen();
+    try {
+      testFactoryWithRootFolder.authenticateAndOpen();
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7555");
```

Comment:
It's better to pass **ex** variable to `fail()` method to have additional info about the catching exception.